vic/drivers/shared_all/src/calc_root_fraction.c

```diff
@@ -62,7 +63,16 @@ calc_root_fractions(veg_con_struct  *veg_con,
         Zsum = 0;
         zone = 0;
 
+        max_itter = -1;
         while (zone < options.ROOT_ZONES) {
+            max_itter++;
+            if (max_itter > 9999) {
```

Comment:
Replace `9999` by some `#define` or constant, e.g. `MAX_ITER`. Any reason why this is set to `9999` that seems a large number of iterations.